### PR TITLE
Feature/dependabot 

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

**Comment by oleorhagen** [comment]
`GOMODULES` needs to be turned off here. That's the problem it seems, according to me and @mzedel recent disussion here

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	github.com/mendersoftware/mendertesting v0.0.0-20190301095837-dedd2c0a90a4
+	github.com/mendersoftware/scopestack v0.0.0-20180403075023-2ce74757611b
+	github.com/pkg/errors v0.0.0-20160916110212-a887431f7f6e
+	github.com/pmezard/go-difflib v1.0.0
+	github.com/remyoudompheng/go-liblzma v0.0.0-20190506200333-81bf2d431b96
+	github.com/sirupsen/logrus v0.0.0-20180329225952-778f2e774c72
+	github.com/stretchr/objx v0.1.0
+	github.com/stretchr/testify v0.0.0-20190109162356-363ebb24d041
+	github.com/ungerik/go-sysfs v0.0.0-20190613143942-7f098ddb67a6
+	github.com/urfave/cli v1.21.0
```
Comment (oleorhagen) [inline go.mod]: Also, this is wrong, as we are using our own `urfave/cli` fork, but with the regular `urfave/cli` import path. That's what tripped the migration off here.

Comment (mzedel) [reply go.mod]: thanks for pointing that out! I was not aware that this would be in there... should now be fixed 👌

**Comment by oleorhagen** [comment]
@mzedel  status here?

**Comment by mzedel** [comment]
ping @lluiscampos - to check if the requested changes were addressed... however I _think_ we wanted to wait with this and go modules for the 2.4 release...

**Comment by mzedel** [comment]
@oleorhagen any idea what the failures might be caused by?

**Comment by coreycothrum** [comment]
> @oleorhagen any idea what the failures might be caused by?

I didn't use '--signoff' when committing, but rather added that line manually to my commit. Maybe that causes the check to fail?

**Comment by oleorhagen** [comment]
> > @oleorhagen any idea what the failures might be caused by?
> 
> I didn't use '--signoff' when committing, but rather added that line manually to my commit. Maybe that causes the check to fail?

? wrong PR ?

**Comment by coreycothrum** [comment]
> > > @oleorhagen any idea what the failures might be caused by?
> > 
> > 
> > I didn't use '--signoff' when committing, but rather added that line manually to my commit. Maybe that causes the check to fail?
> 
> ? wrong PR ?

sorry, yes

**Comment by mzedel** [comment]
@oleorhagen due to the meta-mender connection I would leave the next steps faithfully to you 😛

**Comment by oleorhagen** [comment]
Build: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/152841685

**Comment by kacf** [comment]
@mzedel: Careful when merging something which depends on a checksum update, also required merging [this](https://github.com/mendersoftware/meta-mender/pull/1000).

**Comment by merlin-northern** [comment]
> @mzedel: Careful when merging something which depends on a checksum update, also required merging [this](https://github.com/mendersoftware/meta-mender/pull/1000).

thanks man!

**Comment by mzedel** [comment]
Whoops, sorry! When @oleorhagen mentioned the _dance_ with meta mender I thought running the pipeline was enough - I'll make sure it won't happen again...

**Pull request merged**